### PR TITLE
Support OS install/verification automation in case 0000

### PR DESCRIPTION
**Pull request by JuneZhou19**
@InfraSIM/infrasim_dev 
Currently ESXi, CentOS, CoreOS installation are successful, so adding them as candidate installation OS. Ubuntu installation fails at Downloading debconf preconfiguration file. Redhat installation also didn't succeed.

### CODE REVIEW

**Comment by chenge3** [comment]
@InfraSIM/corecommitters Please help review and merge  the patch, thanks!

**Comment by JuneZhou19** [comment]
It's of too much load for RackHD to install os for 9 nodes together. So we add sleep time between os installation of different nodes.

File: case/interactive/T0000_rackhd_OSInstall.py [hunk 1]
```diff
@@ -56,32 +57,78 @@ def test(self):
...
+                        workflow_obj.cancel()
+                    else:
+                        if i == retry - 1:
+                            self.result(BLOCK, "There is {0} active workflow(s) on node {1}.\n"
+                                               "You may want to check it with: \n"
+                                               "    GET /workflows/{2}\n"
+                                               "or cancel it with: \n"
+                                               "    PUT /workflows/{2}/action".
+                                        format(len(rsp), node_id, workflow_instance_id))
+                            del node_dic[node_id]
```
Comment (turtle-fly) [inline case/interactive/T0000_rackhd_OSInstall.py]: Block or do OS install only on free nodes?

**Comment by turtle-fly** [comment]
@JuneZhou19 @chenge3 and ... you can git-duet for these contribution 😃

**Comment by JuneZhou19** [comment]
@turtle-fly good comments! Code refined!

**Pull request merged**